src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/customer.html.twig

```diff
@@ -222,3 +234,14 @@
     </td>
   </tr>
 </script>
+
+<script id="js-loading-list-row" type="text/template">
+  <tr class="empty_row js-empty-row">
+    <td colspan="8" class="border-0">
+      <div class="text-center grid-table-empty">
+        <p class="mb-0 mt-2"><i class="material-icons">autorenew</i></p>
+        <p class="mb-2">{{ 'Loading'|trans({}, 'Admin.Global') }}</p>
+      </div>
+    </td>
+  </tr>
+</script>
```

Comment:
Would this be usefull on some other pages ? If yes, that would be cool to add it to components folders to make it reusable using a twig include with parameters, with a generic class + class parameter if you want to select it more precisely

Reply:
I'm not sure it could be used like this in other pages, because it is really dependent on where it will be included (in that case a table with 8 columns..) Besides we don't have much `Loading` templates or now (sadly)

So I'm not sure any other pages use this kind o feature, but i we start generalizing this kind of display I guess the message part could be extracted in a common template (just the div part in that case) But it might be too early or this as it's not use somewhere else for now

Reply:
Too early then, but hope that won't be forgotten